### DISCUSSION TITLE
unable to install tfsec on centos

### DISCUSSION THREAD

**Discussion by Ravish17** [aquasecurity/tfsec#1939] [Help and suppports]
install_linux.sh is giving error on centos.
It seems --ignore-missing is not recognized on centos while validating checksum.

$ curl -s https://raw.githubusercontent.com/aquasecurity/tfsec/master/scripts/install_linux.sh | bash
arch=amd64
remote_filename=tfsec-linux-amd64
local_filename=tfsec

====================================================
Looking up the latest version...
Downloading tfsec v1.28.1
Downloaded successfully
Validate checksum
sha256sum: unrecognized option '--ignore-missing'
Try 'sha256sum --help' for more information.
Failed to verify checksum

**Comment by dirsigler** [comment]
I don't think this is an topic for discussion but more an issue.
If the issue still exists please raise it on the [issue section ](https://github.com/aquasecurity/tfsec/issues) of this repository with the system and error details.
